packages/entity-list/src/components/ListView/ListView.js

```diff
@@ -118,7 +119,8 @@ ListView.propTypes = {
   sorting: PropTypes.arrayOf(PropTypes.shape({
     field: PropTypes.string,
     order: PropTypes.string
-  }))
+  })),
+  columnPreferences: PropTypes.objectOf(PropTypes.bool)
```

Comment:
i would either rename this prop/state to columnDisplayPreferences (or something like that) oder change the structure like this:
```
{
  "colnameX": {"display": true},
  "colnameY": {"display": true},
}

```
otherwise the name is a bit confusing for an object of booleans.